Sources/Fullscreen/ExploreDetail/ExploreDetailView.swift

```diff
@@ -59,6 +59,7 @@ public final class ExploreDetailView: UIView {
     }
     private let imageCache = ImageMemoryCache()
     private let layoutBuilder = ExploreDetailLayoutBuilder(elementKind: UICollectionView.elementKindSectionHeader)
+    private let heroViewHeight: CGFloat = 220
```

Comment:
Is the hero view content always such that it's okay to set it to a constant height?

Reply:
yes, the content is pretty static, so the height has always been 220 👍